wire-ios-cryptobox/WireCryptobox/EncryptionSessionsDirectory.swift

```diff
@@ -731,12 +731,8 @@ public struct EncryptionSessionIdentifier: Hashable, Equatable {
         self.domain = String()
     }
 
-    public var hashValue: Int {
-        return self.rawValue.hashValue
-    }
-
     public func hash(into hasher: inout Hasher) {
```

Comment:
Can't this be removed as well?

Reply:
Nope! We need to use that. I wanted to get read of the warning aka remove the property.